src/main/clojure/jarl/builtins/utils.clj

```diff
@@ -0,0 +1,33 @@
+(ns jarl.builtins.utils
```

Comment:
Perhaps we should consolidate this with `jarl.util`? Unless we see this as only useful for built-ins.

Reply:
Something to refactor later, perhaps.

Reply:
Yep!